fnet/cli/predict.py

```diff
@@ -93,6 +93,7 @@ def item_from_dataset(
     else:
         item = dataset[idx]
     target = None
+
```

Comment:
In the future, I'd prefer if we do not arbitrarily add whitespaces to the files with no functional additions.